### DISCUSSION TITLE
Ignore dependency's wasm_bindgen annotations

### DISCUSSION THREAD

**Discussion by kev1n-peters** [wasm-bindgen/wasm-bindgen#2790] [Q&A] [answered]
One of my package's dependencies added wasm_bindgen annotations and now these bindings are being included in the wasm bundle. Is there a way to disable these bindings from being forwarded into the wasm without having to remove the wasm_bindgen annotations from that dependency's source code? Thanks.

Edit: To clarify, I only want bindings to be generated for my code that I annotated with wasm_bindgen.

**Comment by alexcrichton** [comment, marked as answer]
Currently there is no way to ignore the annotations in a dependency, the dependency would need to ffer something like a compile-time feature with `#[cfg_attr(feature = "the_feature", wasm_bindgen)]` gates or similar.
